cime_config/namelist_definition_blom.xml

```diff
@@ -3759,6 +3755,104 @@
     </desc>
   </entry>
 
+  <entry id="ocn_co2_type" modify_via_xml="OCN_CO2_TYPE">
+    <type>char</type>
+    <category>config_bgc</category>
+    <group>bgcnml</group>
+    <values>
+      <value>$OCN_CO2_TYPE</value>
+    </values>
+    <desc>Determines provenance of atmospheric CO2 for gas flux computation.</desc>
+  </entry>
+
+  <!-- hamocc configuration -->
+
```

Comment:
Dear @mvertens , if I understand the approach correct, the entry ids for `use_Agg` and `use_BOXATM` are missing.

Reply:
@jmaerz - thanks for catching this. I think we need to add two more tests for use_BOXATM and use_AGG.
I'll do that today.

Reply:
So in looking at the original buildcpp - BOXATM and AGG were never defined - and I only introduced namelists that corresponded to those variables. That was clearly an oversight on my part. Do we want to test turning these on with hamocc given that they were not defined as part of NorESM configurations before?

Reply:
Hi Mariana, my suspicion is that both cases are not well supported, but I would hope that particularly `AGG` isn't broken (it was used the last time in a study by @JorgSchwinger). `AGG` should work within a coupled NorESM setup, while I have limited idea about `BOXATM` - I would need to look into it - I would suggest to skip the testing for `BOXATM` now, but I feel it would be good to enable switching it on via the xml. I hope to get through your changes by the end of the day.

Reply:
@jmaerz - use_BOXATM and use_AGG are only turned on via user_nl_blom right now - no longer throught the xml since they are no longer CPP variables. I will test that use_AGG works and add a test today.

Reply:
Hi @mvertens , I just realize that `use_FB_BGC_OCE` might also be missing?

Reply:
BOXATM, AGG, FB_BGC_OCN all are options that have not been used quite some time. So currently, these are only for expert users who can activate these options via namelist switches if they know what they are doing.

How much it makes sense to include tests for these options I don't know, I wouldn't see this as a priority.

Reply:
@JorgSchwinger @jmaerz - I have added missing namelists use_boxatm, use_agg and use_fb_bgc_oce to the new namelist group config_bgc. Given the last statement by @JorgSchwinger - can we test these options after this PR and add appropriate tests if everyone agrees? Otherwise - I'm happy to test it in this PR.

Reply:
Hi @mvertens , from my side, I am totally fine to proceed as you suggested. The testing feature is different from the nml configuration.